package.json

```diff
@@ -6,10 +6,11 @@
         "@nomiclabs/hardhat-waffle": "^2.0.2",
         "chai": "^4.3.4",
         "ethereum-waffle": "^3.4.0",
-        "ethers": "^5.5.3",
+        "ethers": "^5.0.0",
         "hardhat": "^2.8.3",
         "prettier": "^2.5.1",
-        "prettier-plugin-solidity": "^1.0.0-beta.19"
+        "prettier-plugin-solidity": "^1.0.0-beta.19",
+        "typechain": "^7.0.0"
```

Comment:
Just making a note that this dependency seemed to do something funky with the dependency chain. Idk if you needed it locally but I've had no problem running without it